src/containers/BooksList.js

```diff
@@ -1,14 +1,23 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import Book from '../components/Book';
-import { removeBook } from '../actions';
+import { removeBook, changeFilter } from '../actions';
+import CategoryFilter from '../components/CategoryFilter';
 
 const BooksList = state => {
   const handleRemoveBook = id => {
     state.removeBook(id);
```

Comment:
**[Required]** Even though the component is working correctly. We generally pass **props** to components, not the **state**. Please replace all.